packages/phpstan-rules/src/Rules/NoVoidGetterMethodRule.php

```diff
@@ -55,11 +55,15 @@ public function process(Node $node, Scope $scope): array
             return [];
         }
 
-        if (! $this->simpleNameResolver->isName($node, 'get*')) {
+        if ($classMethod->isAbstract()) {
+            return [];
+        }
```

Comment:
this is the only new code. everything else is a diff of renaming `$node` -> `$classMethod`